projects/angular/src/utils/list-key-navigation/providers/key-navigation-list.config.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { InjectionToken, ValueProvider } from '@angular/core';
+
+export const ClrListKeyNavigationConfigToken = new InjectionToken('ClrListKeyNavigationConfig');
+
+export const ClrListKeyNavigationConfig = {
```

Comment:
1. These constants should be renamed to snake case for consistency with other tokens/providers.
2. The provider constant should be named as such for consistency with other providers (end with "provider").